cmd/trace-agent/config/testdata/vector_override.yaml

```
@@ -0,0 +1,6 @@
+api_key: api_key_test
+dd_url: https://app.datadoghq.com
+site: datadoghq.eu
+vector.traces:
```

```suggestion
vector
  traces:
```
Shouldn't it be like this? Even if it's the same thing, I think it's more correct as an example.